Dockerfile

```diff
@@ -10,4 +10,6 @@ RUN pip install --no-cache-dir --upgrade -r /src/requirements.txt
 
 COPY ./datahub /src/app
 
+COPY .run.py /src/run.py
+
```

Comment:
Turns out this `run.py` isn't necessary because the changes you made to the config alone fixed the problem.

```suggestion
```